test/unit/api/properties.js

```diff
@@ -77,7 +77,7 @@ describe('api/properties', function () {
         let elem = create(properties.boolean());
         elem.setAttribute('test', value);
```

Comment:
This test doesn't check what it suppose to check, I think. It's not in the changes, it has been here before, but still.
When we are doing `[undefined, null, false, 0, '', 'something'].forEach(` the first thing that we do is `value = JSON.stringify(value);`, which doesn't make sense for the test here because all that we check now is setting string for an attribute, not the different zero-values.